core/arch/arm/plat-stm32mp1/drivers/stm32mp1_rcc.h

```diff
@@ -530,6 +530,11 @@
 #define RCC_MP_AHB5LPENSETR_RNG1LPEN		BIT(6)
 #define RCC_MP_AHB5LPENSETR_BKPSRAMLPEN		BIT(8)
 
+/* RCC_MP_TZAHB6EN(SET|CLR)R bit fields */
+#define RCC_MP_TZAHB6ENSETR_MDMA_POS		0
+#define RCC_MP_TZAHB6ENSETR_MDMA		\
+					BIT(RCC_MP_TZAHB6ENSETR_MDMA_POS)
```

Comment:
Would look better with one less tab I suppose (I mean, remove one tab before the `0` and align `BIT(...` with the `0`).